server/data/interfaces/prisonApiClient.ts

```diff
@@ -31,7 +31,7 @@ import { FullStatus } from '../../interfaces/prisonApi/fullStatus'
 import { SentenceSummary } from '../../interfaces/prisonApi/sentenceSummary'
 import { OffenderIdentifier } from '../../interfaces/prisonApi/offenderIdentifier'
 import { StaffRole } from '../../interfaces/prisonApi/staffRole'
-import { AgenciesEmail, AgencyLocationDetails } from '../../interfaces/prisonApi/agencies'
+import { AgenciesEmail, AgencyDetails } from '../../interfaces/prisonApi/agencies'
```

Comment:
Sorry I renamed this so there are a number of files touched where the only change is just a rename